rust/stackablectl/src/cmds/release.rs

```diff
@@ -84,6 +93,25 @@ pub struct ReleaseInstallArgs {
     local_cluster: CommonClusterArgs,
 }
 
+#[derive(Debug, Args)]
+pub struct ReleaseUpgradeArgs {
+    /// Upgrade to the specified release
+    #[arg(name = "RELEASE")]
+    release: String,
+
+    /// List of product operators to upgrade
+    #[arg(short, long = "include", group = "products")]
+    included_products: Vec<String>,
+
+    /// Blacklist of product operators to install
+    #[arg(short, long = "exclude", group = "products")]
+    excluded_products: Vec<String>,
+
+    /// Namespace in the cluster used to deploy the operators
+    #[arg(long, default_value = DEFAULT_OPERATOR_NAMESPACE, visible_aliases(["operator-ns"]))]
+    pub operator_namespace: String,
```

Comment:
Just a note: We might want to handle the case of:

- install release 24.11 with operator_namespace = "ns1"
- upgrade release 25.3 with operator_namespace = "ns2"

Because:
- The CRDs are cluster scopes, so they would be overwritten, but...
- The operators in the old NS will remain.

I'm fine with us only supporting upgrading in the same namespace (moving namespaces should probably be an uninstall and install). So I guess we'd just need to search for operators in any namespace (labels might be the safest way for detecting).